src/main/resources/logback-spring.xml

```diff
@@ -42,12 +42,41 @@
         <appender-ref ref="STDOUT"/>
     </appender>
 
+    <if condition='"TRUE".equals("${ENABLE_AUDIT_APPENDER}")'>
+        <then>
+            <appender name="KAFKA_AUDIT" class="com.github.danielwegener.logback.kafka.KafkaAppender">
+                <encoder>
+                    <pattern>%msg%n</pattern>
+                </encoder>
+
+                <topic>${KAFKA_AUDIT_TOPIC}</topic>
+
+                <producerConfig>bootstrap.servers=${KAFKA_BINDER_BROKER}</producerConfig>
+                <producerConfig>security.protocol=${KAFKA_CONFIG_SECURITY_PROTOCOL:-SASL_SSL}</producerConfig>
+                <producerConfig>sasl.mechanism=${KAFKA_CONFIG_SASL_MECHANISM:-PLAIN}</producerConfig>
+                <producerConfig>sasl.jaas.config=${KAFKA_AUDIT_PRODUCER_SASL_JAAS_CONFIG}</producerConfig>
+                <producerConfig>client.id=auth</producerConfig>
+                <producerConfig>retries=${KAFKA_AUDIT_RETRIES:-3}</producerConfig>
+                <producerConfig>request.timeout.ms=${KAFKA_AUDIT_REQUEST_TIMEOUT_MS:-60000}</producerConfig>
+                <producerConfig>block.on.buffer.full=false</producerConfig>
+            </appender>
+
+            <logger name="KAFKA_LOGGER" level="INFO" additivity="false">
+                <appender-ref ref="KAFKA_AUDIT"/>
+            </logger>
+        </then>
+        <else>
+            <logger name="KAFKA_LOGGER" level="INFO" additivity="false">
+                <appender-ref ref="ASYNC_STDOUT"/>
+            </logger>
+        </else>
+    </if>
+
     <logger name="AUDIT" additivity="false">
         <appender-ref ref="AUDIT"/>
```

Comment:
here you should define the dependency to KAFKA_AUDIT
ps: we could have more than 1 appender-ref, let's use both

pps: maybe we could be interest on turn-off the KAFKA_AUDIT, we could add here an if-else statement to configure or not the KAFKA_AUDIT appender to AUDIT logger reasoning on a ENV variable `ENABLE_KAFKA_AUDIT_APPENDER`